src/services/has-many-getter.js

```diff
@@ -38,20 +46,20 @@ function HasManyGetter(model, association, opts, params) {
         as: params.associationName,
         scope: false,
         required: false,
-        where,
+        where: getWhere(),
```

Comment:
[MISC] This function returns a promise, the `where` operator expects an object not a promise. This won't work.